tools/macros/macros.xml

```diff
@@ -77,4 +77,34 @@
         #end if
 	</token>
 
+    <token name="@HELP_AUTHORS@">
```

Comment:
You can also add the `<![CDATA[ ... ]]>` things here

Reply:
And around your `@COMMAND` token too

Reply:
Ho ok  it is not done in your macro of xcms and all so I did it like this :) 
So I can add `<![CDATA[ my_things ]]>` in my macro directly ? Even if I put it in my wrapper also ?